test/common/testlib.py

```diff
@@ -2166,6 +2166,20 @@ def write_file(self, path: str, content: str, append: bool = False, owner: Optio
         self.restore_file(path, post_restore_action=post_restore_action)
         m.write(path, content, append=append, owner=owner, perm=perm)
 
+    def reboot(self, timeout_sec: Optional[int] = None):
+        self.allow_restart_journal_messages()
+        if timeout_sec is None:
+            self.machine.reboot()
+        else:
+            self.machine.reboot(timeout_sec=timeout_sec)
+
+    def wait_reboot(self, timeout_sec: Optional[int] = None):
+        self.allow_restart_journal_messages()
+        if timeout_sec is None:
+            self.machine.reboot()
```

Comment:
This is wrong (copy pasta?), needs to be `.wait_reboot()`.

However, do we really need to wrap *both* functions here? This PR introduces reboot() without ever actually calling it.